oai_agents/common/arguments.py

```diff
@@ -20,6 +20,8 @@ def get_arguments(additional_args=[]):
     
     parser.add_argument('--lr', type=float, default=0.001, help='learning rate used in imitation learning. lr for rl is defined in rl.py')
     parser.add_argument('--batch-size', type=int, default=32, help='batch size used in imitation learning. bs for rl is defined in rl.py')
+    parser.add_argument('--seed', type=int, default=48, help='seed used to define self-play neural network')
+    parser.add_argument('--h-dim', type=int, default=256, help='hidden dimension used to define self-play neural network')
```

Comment:
These two new arguments have vague names. If we are only using them when calling SP, they should be called:
`SP-seed, SP-h-dim`

When we create a population of agents we also use self-play agents, but they don't use these seed and h_dims, they use a function called `generate_hdim_and_seed`. If you also want to have more control over those seed, h_dims, you should add a new argument like `POP-seed`, `POP-h-dim`

And the same thing for `FCP`, `FCPWSP`, `SPWSP`, etc.

Reply:
they are used in all functions in train_helper.py

Reply:
Do you have any suggested names?

Reply:
I think our argument variables should all be set in the `set_input` function in `train_agents.py`  so that whenever we want to run a code, we only have to change variables in the `train_agents.py` file and also we can easily track what arguments are being used just by looking at the `set_input` function.

However, the way `args.seed` is implemented right now, we have to go to `train_helper.py` and look at different functions to figure out what seed/hdims are being used. Moreover, I think we should have different arguments for seed and h_dims for each function. Because, right now we know what seeds our FCP, FCPWSP, SPWSP functions work with and they are all different, but the seed that you selected as default has only been tested with SP. 

I suggest adding these arguments:
```
SP-seed, SP-hdim
FCP-seed, FCP-hdim
FCPWSP-seed, FCPWSP-hdim
SPWSP-seed, SPWSP-hdim
```
And setting their coresponding values in `set_input` function in `train_agents.py` and deleting them from `train_helper.py`